sct-commons/src/main/java/org/lfenergy/compas/sct/commons/util/LDeviceStatus.java

```diff
@@ -14,6 +14,6 @@
  */
 @UtilityClass
 public class LDeviceStatus {
-    public static final String ON = "on";
-    public static final String OFF = "off";
+    public static final String ON = LnStatus.ON;
```

Comment:
either you use the String "on" and "on" or you use the same classe for LD and LN but you can not refer to the LnStatus in LDeviceStatus

Reply:
Actually, I reverted this change because I remove LnStatus class due to a comment from Aliou.